itest/send_test.go

```diff
@@ -0,0 +1,75 @@
+package itest
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/lightninglabs/taro/chanutils"
+	"github.com/lightninglabs/taro/tarorpc"
+	"github.com/lightningnetwork/lnd/lntest/wait"
+	"github.com/stretchr/testify/require"
+)
+
+// testBasicSend tests that we can properly send assets back and forth between
+// nodes.
+func testBasicSend(t *harnessTest) {
+	// First, we'll make an normal assets with enough units to allow us to
+	// send it around a few times.
+	rpcAssets := mintAssetsConfirmBatch(
+		t, t.tarod, []*tarorpc.MintAssetRequest{simpleAssets[0]},
+	)
+
+	genBootstrap := rpcAssets[0].AssetGenesis.GenesisBootstrapInfo
+
+	ctxb := context.Background()
+
+	// Now that we have the asset created, we'll make a new node that'll
+	// serve as the node which'll receive the assets.
+	bob := t.lndHarness.NewNode(t.t, "bob", lndDefaultArgs)
```

Comment:
We could use `t.lndHarness.Bob` that already exists to speed things up. Then we only need to call `setupTarodHarness(t.t, t, t.lndHarness.BackendCfg, t.lndHarness.Bob, t.universeServer)`.